lib/spack/spack/relocate.py

```diff
@@ -86,6 +98,24 @@ def get_relative_rpaths(path_name, orig_dir, orig_rpaths):
     return rel_rpaths
 
 
+def get_placeholder_rpaths(path_name, orig_rpaths):
+    """
+    Replaces orig_dir with relative path from dirname(path_name) if an rpath
+    in orig_rpaths contains orig_path. Prefixes $ORIGIN
```

Comment:
Sorry I'm having trouble seeing where `$ORIGIN` gets prefixed here

Reply:
Copied and pasted without changing docstring. Will update.